docs/Webpack.md

```diff
@@ -3,11 +3,11 @@ id: webpack
 title: Using with webpack
 ---
 
-Jest can be used in projects that use [webpack](https://webpack.js.org/) to manage assets, styles, and compilation. webpack _does_ offer some unique challenges over other tools because it integrates directly with your application to allow managing stylesheets, assets like images and fonts, along with the expansive ecosystem of compile-to-JavaScript languages and tools.
+Jest can be used in projects that use [webpack](https://webpack.js.org/) to manage assets, styles, and compilation. webpack _does_ offer some unique benefits over other tools because it integrates directly with your application to allow you the ability to manage assets like images, fonts, and stylesheets, and a host of compile-to-JavaScript languages and tools as part of webpack's expansive ecosystem. 
```

Comment:
I think the original meaning here is that it's harder to integrate a webpack project with Jest than, say, a plain Node project _because_ webpack is used to manage so many things that aren't JS. I'm fine with this, though 🙂 

These benefits aren't really "unique" to webpack though, as Rollup, Parcel, Metro (and probably more?) all support the same thing.

Could we angle this as "webpack, like most bundlers, does a lot, so you might need to configure jest more than you'd have to for more plain JS projects" instead? Not sure about the exact wording...

Reply:
@SimenB I see what you're saying, those mentioned benefits aren't specific to webpack itself, but instead to bundlers in general. Yeah! I like that angle.
Right now I’m working on a draft with your suggestions and I'll post it as soon as it’s finished.